commerce-account-api/src/main/java/com/liferay/commerce/account/constants/CommerceAccountConstants.java

```diff
@@ -23,6 +23,8 @@
 	public static final String ACCOUNT_ADMINISTRATOR_ROLE_NAME =
 		"Account Administrator";
 
+	public static final String ACCOUNT_BUYER_ROLE_NAME = "Buyer";
+
```

Comment:
All of these `*_ROLE_NAME` constants should probably be named `ROLE_NAME_*` instead. Ideally the specifics should come last -- and this actually goes for some of the other constants in this file. Examples of what I mean:

    ACCOUNT_BUYER_ROLE_NAME -> ROLE_NAME_ACCOUNT_BUYER
    ACCOUNT_GROUP_GUEST_NAME -> ACCOUNT_GROUP_NAME_GUEST
    ACCOUNT_ORDER_MANAGER_ROLE_NAME -> ROLE_NAME_ACCOUNT_ORDER_MANAGER
    DEFAULT_PARENT_ACCOUNT_ID -> ACCOUNT_ID_PARENT_DEFAULT (maybe?)
    GUEST_ACCOUNT_ID -> ACCOUNT_ID_GUEST

This type of naming makes it so similar constants are grouped together, which is very much desired in a class like this.